specification/metadata/providertagprefix.md

```diff
@@ -0,0 +1,17 @@
+# Provider Tag Prefix
```

Comment:
Are there scenarios where this is a set of values that providers may use as prefixes? Supporting an array gives more flexibility. We could say its a comma separated string if > 1 and keep it string.

Reply:
Maybe introduce a list type attribute (JSON array)?, which would be here an array of strings.

Reply:
I could definitely see having multiple tag prefixes and would like to see that accounted for.

Reply:
Good point, I will make a suggestion for this.